src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1003,8 +1005,66 @@ static bool addTriggeredSubscriptions_withCache
     //           instead of its std::vector<std::string> ... ?
     //
     StringList aList;
-
-    aList.fill(cSubP->attributes);
+    if (cSubP->onlyChanged && !cSubP->blacklist)
+    {
+      for (unsigned int cavOc = 0; cavOc < modifiedAttrs.size(); ++cavOc)
+      {
+        for (unsigned int avOc = 0; avOc < cSubP->attributes.size(); ++avOc)
+        {
+          if (modifiedAttrs[cavOc] == cSubP->attributes[avOc])
+          {
+            condAttrs.push_back(cSubP->attributes[avOc]);
+          }
+        }
+      }
+      if (condAttrs.size() == 0)
```

Comment:
Style (see https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#m8-code-blocks). It should be:

```
if (condAttrs.size() == 0)
{
  continue;
}
```

Please do a global check. I think there are more similar cases in this PR.

Reply:
fixed in f9887f1